_layouts/google_search.html

```diff
@@ -5,17 +5,46 @@
         <div class="wrapper" id="googleSearchPage">
             <div id="content_col" class="googleSearch">
                 <form id="cse-search-box" action="search.html">
-                    <input type="hidden" name="cx" value="012826289311752768906:ofqwltf6-kg" />
-                    <input type="hidden" name="ie" value="UTF-8" />
                     <div class="formbody">
-                        <input type="text" name="q" id="top_search_searchpage" class="text" value="" placeholder="Search" />
-                        <input type="submit" value="Search" class="submit" >
+                        <input class="text" id="top_search_searchpage" name="q" placeholder="Search" type="text" value="" />
                     </div>
                 </form>
                 <article>
                     <h1>Search results</h1>
-                    <gcse:searchresults-only></gcse:searchresults-only>
+                    <div id="searchResults"></div>
                 </article>
+                <script>
+                    const version = window.location.pathname.split('/')[2];
+                    const queryParams = new URLSearchParams(window.location.search);
+                    const searchInput = document.getElementById('top_search_searchpage');
+                    const searchResults = document.getElementById('searchResults');
+
+                    const displayResults = (results) => {
+                        let html = '<ol>';
+                        if (results.length > 0) {
+                            for (const result of results) {
+                                html += `<li><a href="${result.uri}">${result.title}</a></li>`;
+                            }
+                        } else {
+                            html += `No results found`;
```

Comment:
So if there are no results found, HTML will be `<ol>No results found</ol>`. Don't you think it would be better to omit `<ol>` tags it that case? Just move `let html = '<ol>';` and `html += '</ol>';` into `if`. Or is there a reason why we want to have `<ol>` here? (some styles, maybe)

Reply:
>Don't you think it would be better to omit <ol> tags it that case? 

yeah, makes sense. 

>Or is there a reason why we want to have <ol> here? (some styles, maybe)
nope, ol Is useful in the case of <li> inside, otherwise, it's not a valid HTML structure.